pkgs/applications/audio/jamulus/default.nix

```diff
@@ -0,0 +1,36 @@
+{ mkDerivation, stdenv, fetchFromGitHub, pkgconfig, qtscript, qmake, libjack2
+}:
+
+mkDerivation {
+  pname = "jamulus";
+  version = "3.5.3";
+  src = fetchFromGitHub {
+    owner = "corrados";
+    repo = "jamulus";
+    rev = "5385f4f113884537c18109e01791149754fe96d5";
+    sha256 = "0wrg0g2cligk3pd82h2sh38xgfbdkwklinrk4p0f4iyfiqfzf0ck";
+  };
+
+  nativeBuildInputs = [ pkgconfig qmake ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.